src/util.ts

```diff
@@ -253,7 +253,19 @@ export async function touchTmpFile(path: string): Promise<() => void> {
   return callback;
 }
 
+export function pathEquals(path1: string, path2: string): boolean {
+  if (process.platform !== 'linux') {
+    path1 = path1.toLowerCase();
+    path2 = path2.toLowerCase();
+  }
+  return path1 === path2;
```

Comment:
This issue doesn't occur with macOS and it seems `path.relative` can be used because `path` compares them as case-insensitive in Windows.

<img src="https://user-images.githubusercontent.com/1771005/191570104-9d5d83e9-5bc6-4372-b806-dc83178c39e4.png">

Could you check if below change also work in Windows?

```suggestion
  return path.relative(path1, path2) === '';
```

Reply:
Mac also uses case-insensitive file system, but less problematic. So it will be ok with your suggestion.

I tested this change (with minor change `path` to `upath`). It works in Windows:

```suggestion
  return upath.relative(path1, path2) === '';
```